Unix/installbuilder/datafiles/AIX.data

```diff
@@ -41,6 +41,19 @@ RemoveOmiService() {
 %Preinstall_10
 #include OmiService_funcs
 RemoveOmiService > /dev/null 2> /dev/null
+egrep "^omi:" /etc/group 2>&1 >/dev/null
+if [ $? -ne 0 ]; then
+    echo "Creating omi group ..."
+    mkgroup -a omi
+fi
+grep "/bin/false" /etc/security/login.cfg 2>&1 >/dev/null
+if [ $? -ne 0 ]; then
+   echo "Adding /bin/false as a valid shell..."
+   sed 's?$?,/bin/false?' /etc/security/login.cfg > /tmp/login.cfg
```

Comment:
Ditto for `sed`, `cp`, and `rm`. I won't raise this particular issue for other cases.